ivy/functional/backends/onnx_func/sequence_empty

```diff
@@ -0,0 +1,18 @@
+'''
+Creating an empty tensor sequence with given data type (default is float)
+'''
+
+from onnx import helper
+
+
+class SequenceEmpty:
```

Comment:
Hi! Sorry for the lack of proper instructions for this task. What we need is the backend implementation for `SequenceEmpty`. We just want to add some basic functions that ONNX supports. You can follow [these instructions](https://lets-unify.ai/ivy/contributing/open_tasks.html#ivy-experimental-api) to see what we expect from the PR. Thank you for your interest in helping Ivy! 😊